great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -231,10 +219,11 @@ def __init__(
         Returns:
             None
         """
-        if not BaseDataContext.validate_config(project_config):
-            raise ge_exceptions.InvalidConfigError(
-                "Your project_config is not valid. Try using the CLI check-config command."
-            )
+
+        project_data_context_config: DataContextConfig = (
+            BaseDataContext.validate_config(project_config)
```

Comment:
For static/class methods called in instances, do we want to use the class name or `self`? Good either way but let's be consistent. I think I'm the one who added this class call but I might be the outlier.

Reply:
great question @cdkini. I had a look, and I think it's a mix of class name and self :sweatsmile: My opinion is that we should use class name to distinguish between these and instance methods